phac-nml/irida-next
# Workflow Executions: Fixture and Seed Updates

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Updates the Workflow Executions fixtures and seed data to match default values in `lib/irida/pipeline.rb`

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Ensure that tests pass as expected

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
